Apart from `foxy` and `humble`, other ROS 2 distros are [End of Life](https://docs.ros.org/en/humble/Releases.html). Do we still want CI to pass for those distros?

If so I think I can get `dashing` CI to pass with some `#ifdefine` flags in the code. But I'm not sure why `galactic` CI is failing as seen below.

```
  Found the following packages:
    - realsense2_camera
    - realsense2_camera_msgs
    - realsense2_description
  Package maintainer emails: librs.ros@intel.com
  Traceback (most recent call last):
    File "/tmp/ros_buildfarm/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 287, in resolve_names
      resolved_names = resolve_for_os(
    File "/usr/lib/python3/dist-packages/rosdep2/catkin_support.py", line 87, in resolve_for_os
      d = view.lookup(rosdep_key)
    File "/usr/lib/python3/dist-packages/rosdep2/lookup.py", line 206, in lookup
      return self.rosdep_defs[rosdep_name]
  KeyError: 'ros_workspace'
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "/tmp/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 21, in <module>
      run_module('ros_buildfarm.scripts.devel.create_devel_task_generator', run_name='__main__')
    File "/usr/lib/python3.8/runpy.py", line 210, in run_module
      return _run_code(code, {}, init_globals, run_name, mod_spec)
    File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
      exec(code, run_globals)
    File "/tmp/ros_buildfarm/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 304, in <module>
      sys.exit(main())
    File "/tmp/ros_buildfarm/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 126, in main
      debian_pkg_names += resolve_names(['ros_workspace'], **context)
    File "/tmp/ros_buildfarm/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 290, in resolve_names
      raise RuntimeError(
  RuntimeError: Could not resolve the rosdep key 'ros_workspace'
  ```

--- a/README.md
+++ b/README.md
@@ -288,14 +288,6 @@ Further details on efficient intra-process communication can be found [here](htt
 ### Limitations
 
 * Node components are currently not supported on RCLPY
-* Transformations: `/static_tf` topic will be disabled
-  * To get the transformations published:
-    * Set `tf_publish_rate` to `1.0` in the launch file (or on the command-line, using `-p tf_publish_rate:=1.0`)
-    * Activate and read `/tf` and `/extrinsic/<stream>_to_<stream>` topics
-      * To echo the `/extrinsic/<stream>_to_<stream>` topic you will need to change the default CLI QoS to match the new QoS that the `intra-process` flow uses. E.g.:
-        ```bash
-        ros2 topic echo /extrinsics/depth_to_color --qos-durability=volatile --qos-reliability=reliable
-        ```
 * Compressed images using `image_transport` will be disabled as this isn't supported with intra-process communication
 
 ### Latency test tool and launch file

--- a/realsense2_camera/src/base_realsense_node.cpp
+++ b/realsense2_camera/src/base_realsense_node.cpp
@@ -10,6 +10,13 @@
 #include <fstream>
 #include <image_publisher.h>
 
+// Header files for disabling intra-process comms for static broadcaster.
+#include <rclcpp/publisher_options.hpp>
+// This header file is not available in ROS 2 Dashing.
+#ifndef DASHING
+#include <tf2_ros/qos.hpp>
+#endif
+
 using namespace realsense2_camera;
 
 SyncedImuPublisher::SyncedImuPublisher(rclcpp::Publisher<sensor_msgs::msg::Imu>::SharedPtr imu_publisher, 
@@ -91,11 +98,14 @@ BaseRealSenseNode::BaseRealSenseNode(rclcpp::Node& node,
     {
         ROS_INFO("Intra-Process communication enabled");
     }
-    else
-    {
-        // intra-process requirment of QoS.durability=Volatile cannot be fulfilled with `StaticTransformBroadcaster` as it only support `TransientLocal` durability.
-        _static_tf_broadcaster = std::make_shared<tf2_ros::StaticTransformBroadcaster>(node);
-    }
+
+    rclcpp::PublisherOptionsWithAllocator<std::allocator<void>> options;
+    options.use_intra_process_comm = rclcpp::IntraProcessSetting::Disable;
+    #ifndef DASHING
+    _static_tf_broadcaster = std::make_shared<tf2_ros::StaticTransformBroadcaster>(node, tf2_ros::StaticBroadcasterQoS(), std::move(options));
+    #else
+    _static_tf_broadcaster = std::make_shared<tf2_ros::StaticTransformBroadcaster>(node, rclcpp::QoS(100), std::move(options));
+    #endif
 
     _image_format[1] = CV_8UC1;    // CVBridge type
     _image_format[2] = CV_16UC1;    // CVBridge type

